src/devices_models/devices/hydro_generation.jl

```diff
@@ -147,17 +147,17 @@ function reactivepower_constraints!(
     system_formulation::Type{<:PM.AbstractPowerModel},
     feedforward::Union{Nothing, AbstractAffectFeedForward},
 ) where {H <: PSY.HydroGen, D <: AbstractHydroDispatchFormulation}
-    constraint_data = Vector{DeviceRange}(undef, length(devices))
+    constraints = Vector{DeviceRangeConstraint}(undef, length(devices))
```

Comment:
I would have preferred to keep this as `constraint_data` since the constraint isn't implemented here is just the data. to add it to the model